### PR TITLE
Tonio console

### PR DESCRIPTION
**Pull request by vdorut**
-Traduction des champs utilisateur
-Réinitialiser le mots de passe
-Contexte de carte

### CODE REVIEW

File: ldapadmin/src/main/webapp/console/app/components/user/user.tpl.html [hunk 1]
```diff
@@ -28,93 +28,95 @@
...
               </div>
 
             </div>
 
             <div class="col-md-6">
-              <div class="form-group">
-                <label class="col-sm-4" for="givenName">Password</label> <a>Re-initialiser</a>
+              <div class="form-group form-group-sm">
+                <label class="col-sm-4" for="givenName" translate>user.password</label>
+                <a ng-model ="user.user.mail" href ="/ldapadmin/account/passwordRecovery?email={{user.user.mail | encodeURIComponent}}" translate>user.reset</a>
```
Comment (tonio) [inline ldapadmin/src/main/webapp/console/app/components/user/user.tpl.html]: `ng-model`?

File: ldapadmin/src/main/webapp/console/app/components/user/user.tpl.html [hunk 2]
```diff
@@ -18,7 +18,7 @@
     </ul>
   </div>
 
-  <div class="col-md-10">
+  <div class="col-md-9">
```
Comment (tonio) [inline ldapadmin/src/main/webapp/console/app/components/user/user.tpl.html]: Unexpected ? See `col-md-2` above.

File: ldapadmin/src/main/webapp/console/app/components/user/user.es6 [hunk 3]
```diff
@@ -197,4 +215,6 @@ class UserController {
 UserController.prototype.activate.$inject = [ '$scope' ]
 
 angular.module('admin_console').controller('UserController', UserController)
-
+.filter('encodeURIComponent', () => {
+  return (input) =>  window.encodeURIComponent(input)
+    });
```
Comment (tonio) [inline ldapadmin/src/main/webapp/console/app/components/user/user.es6]: indent

File: ldapadmin/src/main/webapp/console/app/components/user/user.tpl.html [hunk 4]
```diff
@@ -28,93 +28,95 @@
...
                 </div>
               </div>
               <div class="form-group form-group-sm">
-                <label class="col-sm-4" for="preferredLanguage">Contexte</label>
+                <label class="col-sm-4" for="preferredLanguage" translate>user.context</label>
                 <div class="col-sm-8">
-                  <input ng-model="user.user.preferredLanguage" class="form-control" id="preferredLanguage"
-                  placeholder="preferredLanguage">
+                  <select class="form-control" id="contexts" ng-model="user.user.context" >
+                    <option value="">{{user.user.context}}</option>
```
Comment (tonio) [inline ldapadmin/src/main/webapp/console/app/components/user/user.tpl.html]: Wrong label?

File: ldapadmin/src/main/webapp/console/app/components/user/user.tpl.html [hunk 5]
```diff
@@ -28,93 +28,95 @@
...
                 </div>
               </div>
 
             </div>
 
             <div class="col-md-6">
-              <div class="form-group">
-                <label class="col-sm-4" for="givenName">Password</label> <a>Re-initialiser</a>
+              <div class="form-group form-group-sm">
+                <label class="col-sm-4" for="givenName" translate>user.password</label>
```
Comment (tonio) [inline ldapadmin/src/main/webapp/console/app/components/user/user.tpl.html]: Remove `for` attribute

**Comment by vdorut** [comment]
@tonio

**Comment by tonio** [comment]
Merged with https://github.com/georchestra/georchestra/commit/34204d77f2b05a24afb13d6b0a501fbe44687e5a & https://github.com/georchestra/georchestra/commit/dfa05651c0944f02e981e06e503f8603d80c96c6

**Pull request closed without merging**